CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ documentation before upgrading to a new release.
 
 Released closed milestones can be found on [GitHub](https://github.com/Icinga/icinga2/milestones?state=closed).
 
+## 2.12.10 (2023-02-16)
+
+This release updates the bundled Boost and OpenSSL on Windows (#9686)
+and repairs broken SELinux policies (#9689).
```

Comment:
This should be more detailed and state both things fixed, i.e. that on some systems, the policy couldn't be installed, on others, syslog inside checks run with sudo didn't work (if I understood those changes correctly).

Reply:
@htriem Please fix this for me and push on top.

Reply:
As well as in #9692

Reply:
Reworded in both PRs.